scripts/aggregate_reviews.py

```
@@ -42,10 +46,16 @@ def run_aggregation():
     findings_map = defaultdict(list)
     for comment in comments:
         author = comment.get('user', {}).get('login')
+        if not author:
+            continue
         file_path = comment.get('path')
         line = comment.get('line')
 
         current_tool = next((name for name, id in TOOL_IDENTIFIERS.items() if id == author), None)
+        if not current_tool:.
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)